main.js

```diff
@@ -0,0 +1,102 @@
+const more = document.querySelector('.btn-more');
+const feature = document.querySelector('.feature-cards');
+
+const team = [
+  {
+    name: 'Obed Bamfo',
+    location: 'United States of America',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam cupiditate ex esse
+    sapiente. Distinctio quae adipisci ipsum sed nihil iste tenetur, minus laboriosam,
+    quos animi eligendi eos, dignissimos perferendis nesciunt?`,
+    image: ['img/five.jpeg'],
+  },
+  {
+    name: 'Pesus Darko',
+    location: 'United States of America',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam cupiditate ex esse
+    sapiente. Distinctio quae adipisci ipsum sed nihil iste tenetur, minus laboriosam,
+    quos animi eligendi eos, dignissimos perferendis nesciunt?`,
+    image: ['img/five.jpeg'],
+  },
+  {
+    name: 'Evans Bamfo',
+    location: 'United States of America',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam cupiditate ex esse
+    sapiente. Distinctio quae adipisci ipsum sed nihil iste tenetur, minus laboriosam,
+    quos animi eligendi eos, dignissimos perferendis nesciunt?`,
+    image: ['img/five.jpeg'],
```

Comment:
- [ ] Please 🙏🏽 add different images for each member in the "Performer" section.